backend/add_item.py

```diff
@@ -5,14 +5,33 @@
 from datetime import datetime
 import mongoengine.errors
 from utils import Error
+import base64
+from os import environ
 
 add_item_blueprint = Blueprint("add_item", __name__)
 
 
+def add_image(image_b64, item):
```

Comment:
Since the file is called add_item, it would be cleaner to remove the add_image function from this class and instead move to a separate file called add_image.

Reply:
The files are separated to make one file per API endpoint. For example, this add_item file provides the logic for the POST /item route used by the frontend. add_image is just a helper function that does part of the work for the same route. Thus, I think it would be best if it stays in the same file. Otherwise, we would make a new file for each function by the same logic, and one file per function would be too many files.

I understand that the file name add_item may have been confusing, since it matches the name of the route function instead of the API endpoint. However, the API endpoint names are shorter ("item" for this route), so changing the file names to match endpoint names would probably be worse (because "item.py" is much less clear than "add_item.py").